mock/docs/site-defaults.cfg

```diff
@@ -186,6 +186,18 @@
 #config_opts['bootstrap_chroot_additional_packages'] = []
 #config_opts['bootstrap_module_setup_commands'] = []
 
+# The following buildroot-image options are internal to Mock (at least for
+# now).  Please use the --buildroot-image option instead, and avoid setting
+# these in Mock configuration files.  The semantics are similar to the
```

Comment:
If we do not recommend to change it, can this be moved below line 397?

```
# Things that you can change, but we dont recommend it:
```

Reply:
Done.